.github/workflows/codeql.yml

```
@@ -0,0 +1,40 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+  schedule:
+    - cron: '31 11 * * 4'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
```

We have some `typescript` as well, right?

```suggestion
        language: [ 'javascript', 'typescript' ]
```

---

I would think that Typescript is already scanned with javascript language, see https://codeql.github.com/docs/codeql-overview/supported-languages-and-frameworks/#languages-and-compilers 
I it saying:
> TypeScript analysis is performed by running the JavaScript extractor with TypeScript enabled. This is the default for LGTM.